<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Materiales y métodos | A Minimal Book Example</title>
  <meta name="description" content="<p>This is a minimal example of using the bookdown package to write a book.
The HTML output format for this example is bookdown::gitbook,
set in the _output.yml file.</p>" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Materiales y métodos | A Minimal Book Example" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="<p>This is a minimal example of using the bookdown package to write a book.
The HTML output format for this example is bookdown::gitbook,
set in the _output.yml file.</p>" />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Materiales y métodos | A Minimal Book Example" />
  
  <meta name="twitter:description" content="<p>This is a minimal example of using the bookdown package to write a book.
The HTML output format for this example is bookdown::gitbook,
set in the _output.yml file.</p>" />
  

<meta name="author" content="John Doe" />


<meta name="date" content="2021-11-08" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="antecedentes.html"/>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Minimal Book Example</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Resumen</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#summary"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#agradecimientos"><i class="fa fa-check"></i>Agradecimientos</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#tabla-de-contenido"><i class="fa fa-check"></i>Tabla de contenido</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="índice-de-cuadros-y-figuras.html"><a href="índice-de-cuadros-y-figuras.html"><i class="fa fa-check"></i>Índice de cuadros y figuras</a><ul>
<li class="chapter" data-level="" data-path="índice-de-cuadros-y-figuras.html"><a href="índice-de-cuadros-y-figuras.html#figuras"><i class="fa fa-check"></i>Figuras</a></li>
<li class="chapter" data-level="" data-path="índice-de-cuadros-y-figuras.html"><a href="índice-de-cuadros-y-figuras.html#tablas"><i class="fa fa-check"></i>Tablas</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="introducción.html"><a href="introducción.html"><i class="fa fa-check"></i>Introducción</a><ul>
<li class="chapter" data-level="" data-path="introducción.html"><a href="introducción.html#objetivo-general"><i class="fa fa-check"></i>Objetivo general</a></li>
<li class="chapter" data-level="" data-path="introducción.html"><a href="introducción.html#objetivos-particulares"><i class="fa fa-check"></i>Objetivos particulares</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="antecedentes.html"><a href="antecedentes.html"><i class="fa fa-check"></i>Antecedentes</a><ul>
<li class="chapter" data-level="" data-path="antecedentes.html"><a href="antecedentes.html#definición-y-generalidades-de-una-red"><i class="fa fa-check"></i>Definición y generalidades de una red</a></li>
<li class="chapter" data-level="" data-path="antecedentes.html"><a href="antecedentes.html#propiedades-topológicas-de-las-redes"><i class="fa fa-check"></i>Propiedades topológicas de las redes</a></li>
<li class="chapter" data-level="" data-path="antecedentes.html"><a href="antecedentes.html#definición-y-características-de-las-redes-multicapa"><i class="fa fa-check"></i>Definición y características de las redes multicapa</a></li>
<li class="chapter" data-level="" data-path="antecedentes.html"><a href="antecedentes.html#aplicaciones-de-las-redes-multicapa"><i class="fa fa-check"></i>Aplicaciones de las redes multicapa</a></li>
<li class="chapter" data-level="" data-path="antecedentes.html"><a href="antecedentes.html#inferencias-de-redes"><i class="fa fa-check"></i>Inferencias de redes</a></li>
<li class="chapter" data-level="" data-path="antecedentes.html"><a href="antecedentes.html#la-microbiota-intestinal-y-su-impacto-en-la-salud-humana"><i class="fa fa-check"></i>La microbiota intestinal y su impacto en la salud humana</a></li>
<li class="chapter" data-level="" data-path="antecedentes.html"><a href="antecedentes.html#uso-de-redes-en-la-exploración-de-la-microbiota"><i class="fa fa-check"></i>Uso de redes en la exploración de la microbiota</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="materiales-y-métodos.html"><a href="materiales-y-métodos.html"><i class="fa fa-check"></i>Materiales y métodos</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">A Minimal Book Example</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="materiales-y-métodos" class="section level1 unnumbered">
<h1>Materiales y métodos</h1>
<hr />
<p>Previamente hemos analizado las posibilidades y limitaciones de las redes multicapa, así como el procedimiento que implica su inferencia y construcción. Para la realización de este trabajo se utilizó la versión 4.1.0 (2021-05-18) de R. Se emplearon los paquetes <em>igraph</em>, <em>phyloseq</em>, <em>devtools</em>, <em>SpiecEasi</em>, <em>seqtime</em>, <em>muxViz</em> y <em>multinet</em>. Algunos de los paquetes están disponibles en CRAN y pueden ser descargados mediante el comando <em>install.packages()</em> de R, pero otros fueron descargados de repositorios de GitHub o del proyecto para análisis de datos genómicos Bioconductor.</p>
<table>
<colgroup>
<col width="26%" />
<col width="16%" />
<col width="26%" />
<col width="30%" />
</colgroup>
<thead>
<tr class="header">
<th>Paquete</th>
<th>Uso</th>
<th>Fuente</th>
<th>Descarga</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>igraph</td>
<td>Construcción, diseño, análisis y manipulación de redes.</td>
<td>CRAN</td>
<td>install.packages(“igraph”)</td>
</tr>
<tr class="even">
<td>phyloseq</td>
<td>Herramientas para la importación, análisis, almacenamiento y la visualización de datos de microbioma.</td>
<td>Bioconductor</td>
<td>if(!requireNamespace(“BiocManager”, quietly = TRUE)) install.packages(“BiocManager”); BiocManager::install(“phyloseq”)</td>
</tr>
<tr class="odd">
<td>devtools</td>
<td>Provee las funciones necesarias para el desarrollo de paquetes de R y descarga desde repositorios GitHub.</td>
<td>CRAN</td>
<td>install.packages(“devtools”)</td>
</tr>
<tr class="even">
<td>SpiecEasi</td>
<td>Estimación de la covarianza de datos de abundancia e inferencia de redes ecológicas.</td>
<td>GitHub</td>
<td>library(devtools); install_github(“zdk123/SpiecEasi”)</td>
</tr>
<tr class="odd">
<td>seqtime</td>
<td>Provee datos de series temporales de microbiota, y herramientas para simular y analizar la dinámica de comunidades microbianas en el tiempo.</td>
<td>GitHub</td>
<td>library(devtools); install_github(“hallucigenia- sparsa/seqtime”)</td>
</tr>
<tr class="even">
<td>muxViz</td>
<td>Construcción, diseño y análisis de redes multicapa.</td>
<td>GitHub</td>
<td>devtools::install_github(“manlius/muxViz”)</td>
</tr>
<tr class="odd">
<td>multinet</td>
<td>Construcción, diseño y análisis de redes multicapa.</td>
<td>CRAN</td>
<td>install.packages(“multinet”)</td>
</tr>
<tr class="even">
<td>corrplot</td>
<td>Análisis de la correlación de matrices.</td>
<td>CRAN</td>
<td>install.packages(“corrplot”)</td>
</tr>
</tbody>
</table>

<div id="refs" class="references">
<div>
<p>Adeshirlarijaney, A., Zou, J., Tran, H. Q., Chassaing, B., &amp; Gewirtz, A. T. (2019). Amelioration of metabolic syndrome by metformin associates with reduced indices of low-grade inflammation independently of the gut microbiota. <em>American Journal of Physiology-Endocrinology and Metabolism</em>, <em>317</em>(6), E1121–E1130.</p>
</div>
<div>
<p>Albert, R., &amp; Barabási, A.-L. (2002). Statistical mechanics of complex networks. <em>Reviews of Modern Physics</em>, <em>74</em>(1), 47.</p>
</div>
<div>
<p>Aleta, A., Meloni, S., &amp; Moreno, Y. (2017). A multilayer perspective for the analysis of urban transportation systems. <em>Scientific Reports</em>, <em>7</em>(1), 1–9.</p>
</div>
<div>
<p>Aleta, A., &amp; Moreno, Y. (2019). Multilayer networks in a nutshell. <em>Annual Review of Condensed Matter Physics</em>, <em>10</em>, 45–62.</p>
</div>
<div>
<p>Allin, K. H., Tremaroli, V., Caesar, R., Jensen, B. A., Damgaard, M. T., Bahl, M. I., … others. (2018). Aberrant intestinal microbiota in individuals with prediabetes. <em>Diabetologia</em>, <em>61</em>(4), 810–820.</p>
</div>
<div>
<p>Anderson, P. W. (1972). More is different. <em>Science</em>, <em>177</em>(4047), 393–396.</p>
</div>
<div>
<p>Arda, H., Arenas, A., Sharma, A., &amp; others. (2019). The multiplex network of human diseases. <em>NPJ Systems Biology and Applications</em>, <em>5</em>(1).</p>
</div>
<div>
<p>Baas, N. (1994). Emergence, hierarchies and hyperstructures. Artifidal ufe, n’Ill, vol. <em>Addison-Wesley</em>, <em>515</em>, 537.</p>
</div>
<div>
<p>Baldassano, S. N., &amp; Bassett, D. S. (2016). Topological distortion and reorganized modular structure of gut microbial co-occurrence networks in inflammatory bowel disease. <em>Scientific Reports</em>, <em>6</em>(1), 1–14.</p>
</div>
<div>
<p>Barrat, A., Barthelemy, M., &amp; Vespignani, A. (2008). <em>Dynamical processes on complex networks</em>. Cambridge university press.</p>
</div>
<div>
<p>Bäckhed, F., Ding, H., Wang, T., Hooper, L. V., Koh, G. Y., Nagy, A., … Gordon, J. I. (2004). The gut microbiota as an environmental factor that regulates fat storage. <em>Proceedings of the National Academy of Sciences</em>, <em>101</em>(44), 15718–15723.</p>
</div>
<div>
<p>Bentley, B., Branicky, R., Barnes, C. L., Chew, Y. L., Yemini, E., Bullmore, E. T., … Schafer, W. R. (2016). The multilayer connectome of caenorhabditis elegans. <em>PLoS Computational Biology</em>, <em>12</em>(12), e1005283.</p>
</div>
<div>
<p>Bianconi, G. (2018). <em>Multilayer networks: Structure and function</em>. Oxford university press.</p>
</div>
<div>
<p>Bonneau, R., Reiss, D. J., Shannon, P., Facciotti, M., Hood, L., Baliga, N. S., &amp; Thorsson, V. (2006). The inferelator: An algorithm for learning parsimonious regulatory networks from systems-biology data sets de novo. <em>Genome Biology</em>, <em>7</em>(5), 1–16.</p>
</div>
<div>
<p>Bryrup, T., Thomsen, C. W., Kern, T., Allin, K. H., Brandslund, I., Jørgensen, N. R., … others. (2019). Metformin-induced changes of the gut microbiota in healthy young men: Results of a non-blinded, one-armed intervention study. <em>Diabetologia</em>, <em>62</em>(6), 1024–1035.</p>
</div>
<div>
<p>Buldyrev, S. V., Parshani, R., Paul, G., Stanley, H. E., &amp; Havlin, S. (2010). Catastrophic cascade of failures in interdependent networks. <em>Nature</em>, <em>464</em>(7291), 1025–1028.</p>
</div>
<div>
<p>Butte, A. J., &amp; Kohane, I. S. (1999a). Mutual information relevance networks: Functional genomic clustering using pairwise entropy measurements. In <em>Biocomputing 2000</em> (pp. 418–429). World Scientific.</p>
</div>
<div>
<p>Butte, A. J., &amp; Kohane, I. S. (1999b). Unsupervised knowledge discovery in medical databases using relevance networks. <em>Proceedings of the Amia Symposium</em>, 711. American Medical Informatics Association.</p>
</div>
<div>
<p>Chen, E. Z., &amp; Li, H. (2016). A two-part mixed-effects model for analyzing longitudinal microbiome compositional data. <em>Bioinformatics</em>, <em>32</em>(17), 2611–2617.</p>
</div>
<div>
<p>Clemente, J. C., Pehrsson, E. C., Blaser, M. J., Sandhu, K., Gao, Z., Wang, B., … others. (2015). The microbiome of uncontacted amerindians. <em>Science Advances</em>, <em>1</em>(3), e1500183.</p>
</div>
<div>
<p>Colizza, V., Flammini, A., Serrano, M. A., &amp; Vespignani, A. (2006). Detecting rich-club ordering in complex networks. <em>Nature Physics</em>, <em>2</em>(2), 110–115.</p>
</div>
<div>
<p>Collins, H. L., Drazul-Schrader, D., Sulpizio, A. C., Koster, P. D., Williamson, Y., Adelman, S. J., … Bellamine, A. (2016). L-carnitine intake and high trimethylamine n-oxide plasma levels correlate with low aortic lesions in apoe-/- transgenic mice expressing cetp. <em>Atherosclerosis</em>, <em>244</em>, 29–37.</p>
</div>
<div>
<p>Cox, L. M., &amp; Blaser, M. J. (2015). Antibiotics in early life and obesity. <em>Nature Reviews Endocrinology</em>, <em>11</em>(3), 182–190.</p>
</div>
<div>
<p>Cozzo, E., Arruda, G. F. de, Rodrigues, F. A., &amp; Moreno, Y. (2016). Multilayer networks: Metrics and spectral properties. In <em>Interconnected networks</em> (pp. 17–35). Springer.</p>
</div>
<div>
<p>Cozzo, E., Kivelä, M., De Domenico, M., Solé, A., Arenas, A., Gómez, S., … Moreno, Y. (2013). Clustering coefficients in multiplex networks. <em>arXiv Preprint arXiv:1307.6780</em>.</p>
</div>
<div>
<p>Cui, X., Ye, L., Li, J., Jin, L., Wang, W., Li, S., … others. (2018). Metagenomic and metabolomic analyses unveil dysbiosis of gut microbiota in chronic heart failure patients. <em>Scientific Reports</em>, <em>8</em>(1), 1–15.</p>
</div>
<div>
<p>Das, A. B. (2020). Small-world networks of prognostic genes associated with lung adenocarcinoma development. <em>Genomics</em>, <em>112</em>(6), 4078–4088.</p>
</div>
<div>
<p>Del Chierico, F., Nobili, V., Vernocchi, P., Russo, A., De Stefanis, C., Gnani, D., … others. (2017). Gut microbiota profiling of pediatric nonalcoholic fatty liver disease and obese patients unveiled by an integrated meta-omics-based approach. <em>Hepatology</em>, <em>65</em>(2), 451–464.</p>
</div>
<div>
<p>Del Genio, C. I., Gómez-Gardeñes, J., Bonamassa, I., &amp; Boccaletti, S. (2016). Synchronization in networks with multiple interaction layers. <em>Science Advances</em>, <em>2</em>(11), e1601679.</p>
</div>
<div>
<p>Donges, J. F., Schultz, H. C., Marwan, N., Zou, Y., &amp; Kurths, J. (2011). Investigating the topology of interacting networks. <em>The European Physical Journal B</em>, <em>84</em>(4), 635–651.</p>
</div>
<div>
<p>Falkenburg, B., &amp; Morrison, M. (2015). Why more is different. <em>Philosophical Issues in Condensed Matter Physics and Complex Systems. Springer, Berlin, Heidelberg</em>.</p>
</div>
<div>
<p>Falony, G., Vieira-Silva, S., &amp; Raes, J. (2018). Richness and ecosystem development across faecal snapshots of the gut microbiota. <em>Nature Microbiology</em>, <em>3</em>(5), 526–528.</p>
</div>
<div>
<p>Fan, Y., Li, Y., Chen, Y., Zhao, Y.-J., Liu, L.-W., Li, J., … others. (2016). Comprehensive metabolomic characterization of coronary artery diseases. <em>Journal of the American College of Cardiology</em>, <em>68</em>(12), 1281–1293.</p>
</div>
<div>
<p>Fan, Y., &amp; Pedersen, O. (2021). Gut microbiota in human metabolic health and disease. <em>Nature Reviews Microbiology</em>, <em>19</em>(1), 55–71.</p>
</div>
<div>
<p>Farahani, F. V., Karwowski, W., &amp; Lighthall, N. R. (2019). Application of graph theory for identifying connectivity patterns in human brain networks: A systematic review. <em>Frontiers in Neuroscience</em>, <em>13</em>, 585.</p>
</div>
<div>
<p>Flint, A., Raben, A., Rehfeld, J., Holst, J., &amp; Astrup, A. (2000). The effect of glucagon-like peptide-1 on energy expenditure and substrate metabolism in humans. <em>International Journal of Obesity</em>, <em>24</em>(3), 288–298.</p>
</div>
<div>
<p>Friedman, J., &amp; Alm, E. J. (2012). <em>Inferring correlation networks from genomic survey data</em>.</p>
</div>
<div>
<p>Gamiño-Arroyo, A. E., Barrios-Ceballos, M. P., Peña, L. P. C. de la, Anaya-Velázquez, F., &amp; Padilla-Vaca, F. (2005). Flora normal, probióticos y salud humana. <em>Acta Universitaria</em>, <em>15</em>(3), 34–40.</p>
</div>
<div>
<p>Gardner, M. (1970). The fantastic combinations of jhon conway’s new solitaire game’Life. <em>Sc. Am.</em>, <em>223</em>, 20–123.</p>
</div>
<div>
<p>Gomez, S., Diaz-Guilera, A., Gomez-Gardenes, J., Perez-Vicente, C. J., Moreno, Y., &amp; Arenas, A. (2013). Diffusion dynamics on multiplex networks. <em>Physical Review Letters</em>, <em>110</em>(2), 028701.</p>
</div>
<div>
<p>Gophna, U., Konikoff, T., &amp; Nielsen, H. B. (2017). Oscillospira and related bacteria–from metagenomic species to metabolic features. <em>Environmental Microbiology</em>, <em>19</em>(3), 835–841.</p>
</div>
<div>
<p>Górski, A., Bollyky, P. L., Przybylski, M., Borysowski, J., Międzybrodzki, R., Jończyk-Matysiak, E., &amp; Weber-Dąbrowska, B. (2019). Perspectives of phage therapy in non-bacterial infections. <em>Frontiers in Microbiology</em>, <em>9</em>, 3306.</p>
</div>
<div>
<p>Guimera, R., &amp; Amaral, L. A. N. (2005). Cartography of complex networks: Modules and universal roles. <em>Journal of Statistical Mechanics: Theory and Experiment</em>, <em>2005</em>(02), P02001.</p>
</div>
<div>
<p>Hirano, H., &amp; Takemoto, K. (2019). Difficulty in inferring microbial community structure based on co-occurrence network approaches. <em>BMC Bioinformatics</em>, <em>20</em>(1), 1–14.</p>
</div>
<div>
<p>Holme, P., &amp; Saramäki, J. (2012). Temporal networks. <em>Physics Reports</em>, <em>519</em>(3), 97–125.</p>
</div>
<div>
<p>Iacovacci, J., Rahmede, C., Arenas, A., &amp; Bianconi, G. (2016). Functional multiplex pagerank. <em>EPL (Europhysics Letters)</em>, <em>116</em>(2), 28004.</p>
</div>
<div>
<p>Jackson, M. A., Bonder, M. J., Kuncheva, Z., Zierer, J., Fu, J., Kurilshikov, A., … others. (2018). Detection of stable community structures within gut microbiota co-occurrence networks from different human populations. <em>PeerJ</em>, <em>6</em>, e4303.</p>
</div>
<div>
<p>Kanawati, R. (2015). Multiplex network mining: A brief survey. <em>IEEE Intell. Informatics Bull.</em>, <em>16</em>(1), 24–27.</p>
</div>
<div>
<p>Kenett, D. Y., Perc, M., &amp; Boccaletti, S. (2015). Networks of networks–an introduction. <em>Chaos, Solitons &amp; Fractals</em>, <em>80</em>, 1–6.</p>
</div>
<div>
<p>Korem, T., Zeevi, D., Suez, J., Weinberger, A., Avnit-Sagi, T., Pompan-Lotan, M., … others. (2015). Growth dynamics of gut microbiota in health and disease inferred from single metagenomic samples. <em>Science</em>, <em>349</em>(6252), 1101–1106.</p>
</div>
<div>
<p>Kurtz, Z. D., Müller, C. L., Miraldi, E. R., Littman, D. R., Blaser, M. J., &amp; Bonneau, R. A. (2015). Sparse and compositionally robust inference of microbial ecological networks. <em>PLoS Computational Biology</em>, <em>11</em>(5), e1004226.</p>
</div>
<div>
<p>Lancichinetti, A., &amp; Fortunato, S. (2012). Consensus clustering in complex networks. <em>Scientific Reports</em>, <em>2</em>(1), 1–7.</p>
</div>
<div>
<p>Langfelder, P., &amp; Horvath, S. (2008). WGCNA: An r package for weighted correlation network analysis. <em>BMC Bioinformatics</em>, <em>9</em>(1), 1–13.</p>
</div>
<div>
<p>Layeghifard, M., Hwang, D. M., &amp; Guttman, D. S. (2017). Disentangling interactions in the microbiome: A network perspective. <em>Trends in Microbiology</em>, <em>25</em>(3), 217–228.</p>
</div>
<div>
<p>Layeghifard, M., Hwang, D. M., &amp; Guttman, D. S. (2018). Constructing and analyzing microbiome networks in r. In <em>Microbiome analysis</em> (pp. 243–266). Springer.</p>
</div>
<div>
<p>Lazega, E., &amp; Pattison, P. E. (1999). Multiplexity, generalized exchange and cooperation in organizations: A case study. <em>Social Networks</em>, <em>21</em>(1), 67–90.</p>
</div>
<div>
<p>Lewes, G. H. (1891). <em>Problems of life and mind, first series: The foundations of a creed, vol 2.</em></p>
</div>
<div>
<p>Liu, Y.-Y., &amp; Barabási, A.-L. (2016). Control principles of complex systems. <em>Reviews of Modern Physics</em>, <em>88</em>(3), 035006.</p>
</div>
<div>
<p>Liu, Y.-Y., Slotine, J.-J., &amp; Barabási, A.-L. (2011). Controllability of complex networks. <em>Nature</em>, <em>473</em>(7346), 167–173.</p>
</div>
<div>
<p>Lozupone, C., Faust, K., Raes, J., Faith, J. J., Frank, D. N., Zaneveld, J., … Knight, R. (2012). Identifying genomic and metabolic features that can underlie early successional and opportunistic lifestyles of human gut symbionts. <em>Genome Research</em>, <em>22</em>(10), 1974–1984.</p>
</div>
<div>
<p>Ludwig, D., Jones, D. D., &amp; Holling, C. S. (1978). Qualitative analysis of insect outbreak systems: The spruce budworm and forest. <em>The Journal of Animal Ecology</em>, 315–332.</p>
</div>
<div>
<p>Macfarlane, S., &amp; Macfarlane, G. T. (2006). Composition and metabolic activities of bacterial biofilms colonizing food residues in the human gut. <em>Applied and Environmental Microbiology</em>, <em>72</em>(9), 6204–6211.</p>
</div>
<div>
<p>Margolin, A. A., Nemenman, I., Basso, K., Wiggins, C., Stolovitzky, G., Dalla Favera, R., &amp; Califano, A. (2006). ARACNE: An algorithm for the reconstruction of gene regulatory networks in a mammalian cellular context. <em>BMC Bioinformatics</em>, <em>7</em>, 1–15. Springer.</p>
</div>
<div>
<p>Maslowski, K. M., Vieira, A. T., Ng, A., Kranich, J., Sierro, F., Yu, D., … others. (2009). Regulation of inflammatory responses by gut microbiota and chemoattractant receptor gpr43. <em>Nature</em>, <em>461</em>(7268), 1282–1286.</p>
</div>
<div>
<p>Mason, O., &amp; Verwoerd, M. (2007). Graph theory and networks in biology. <em>IET Systems Biology</em>, <em>1</em>(2), 89–119.</p>
</div>
<div>
<p>Meinshausen, N., &amp; Bühlmann, P. (2006). High-dimensional graphs and variable selection with the lasso. <em>The Annals of Statistics</em>, <em>34</em>(3), 1436–1462.</p>
</div>
<div>
<p>Mello, V. D. de, Paananen, J., Lindström, J., Lankinen, M. A., Shi, L., Kuusisto, J., … others. (2017). Indolepropionic acid and novel lipid metabolites are associated with a lower risk of type 2 diabetes in the finnish diabetes prevention study. <em>Scientific Reports</em>, <em>7</em>(1), 1–12.</p>
</div>
<div>
<p>Meyer, P. E., Lafitte, F., &amp; Bontempi, G. (2008). Minet: AR/bioconductor package for inferring large transcriptional networks using mutual information. <em>BMC Bioinformatics</em>, <em>9</em>(1), 1–10.</p>
</div>
<div>
<p>Michos, E. D., McEvoy, J. W., &amp; Blumenthal, R. S. (2019). Lipid management for the prevention of atherosclerotic cardiovascular disease. <em>New England Journal of Medicine</em>, <em>381</em>(16), 1557–1567.</p>
</div>
<div>
<p>Miller, T. L., Wolin, M., Macario, E. C. de, &amp; Macario, A. (1982). Isolation of methanobrevibacter smithii from human feces. <em>Applied and Environmental Microbiology</em>, <em>43</em>(1), 227–232.</p>
</div>
<div>
<p>Newman, M., &amp; Barkema, G. (n.d.). Oxford university press; new york: 2010. <em>Networks: An Introduction.[Google Scholar]</em>.</p>
</div>
<div>
<p>Newman, M. E., Barabási, A.-L. E., &amp; Watts, D. J. (2006). <em>The structure and dynamics of networks.</em> Princeton university press.</p>
</div>
<div>
<p>O’Connor, T. (1994). Emergent properties. <em>American Philosophical Quarterly</em>, <em>31</em>(2), 91–104.</p>
</div>
<div>
<p>Padgett, J. F., &amp; Ansell, C. K. (1993). Robust action and the rise of the medici, 1400-1434. <em>American Journal of Sociology</em>, <em>98</em>(6), 1259–1319.</p>
</div>
<div>
<p>Peng, H., Long, F., &amp; Ding, C. (2005). Feature selection based on mutual information criteria of max-dependency, max-relevance, and min-redundancy. <em>IEEE Transactions on Pattern Analysis and Machine Intelligence</em>, <em>27</em>(8), 1226–1238.</p>
</div>
<div>
<p>Qin, J., Li, R., Raes, J., Arumugam, M., Burgdorf, K. S., Manichanh, C., … others. (2010). A human gut microbial gene catalogue established by metagenomic sequencing. <em>Nature</em>, <em>464</em>(7285), 59–65.</p>
</div>
<div>
<p>Rao, R., Seth, A., &amp; Sheth, P. (2004). Recent advances in alcoholic liver disease i. Role of intestinal permeability and endotoxemia in alcoholic liver disease. <em>American Journal of Physiology-Gastrointestinal and Liver Physiology</em>, <em>286</em>(6), G881–G884.</p>
</div>
<div>
<p>Ridaura, V. K., Faith, J. J., Rey, F. E., Cheng, J., Duncan, A. E., Kau, A. L., … others. (2013). Gut microbiota from twins discordant for obesity modulate metabolism in mice. <em>Science</em>, <em>341</em>(6150).</p>
</div>
<div>
<p>Rothschild, D., Weissbrod, O., Barkan, E., Kurilshikov, A., Korem, T., Zeevi, D., … others. (2018). Environment dominates over host genetics in shaping human gut microbiota. <em>Nature</em>, <em>555</em>(7695), 210–215.</p>
</div>
<div>
<p>Russell, W. R., Duncan, S. H., Scobbie, L., Duncan, G., Cantlay, L., Calder, A. G., … Flint, H. J. (2013). Major phenylpropanoid-derived metabolites in the human gut can arise from microbial fermentation of protein. <em>Molecular Nutrition &amp; Food Research</em>, <em>57</em>(3), 523–535.</p>
</div>
<div>
<p>Sanz, J., Xia, C.-Y., Meloni, S., &amp; Moreno, Y. (2014). Dynamics of interacting diseases. <em>Physical Review X</em>, <em>4</em>(4), 041005.</p>
</div>
<div>
<p>Schaeffer, S. E. (2007). Graph clustering. <em>Computer Science Review</em>, <em>1</em>(1), 27–64.</p>
</div>
<div>
<p>Schleuning, M., Neuschulz, E. L., Albrecht, J., Bender, I. M., Bowler, D. E., Dehling, D. M., … others. (2020). Trait-based assessments of climate-change impacts on interacting species. <em>Trends in Ecology &amp; Evolution</em>, <em>35</em>(4), 319–328.</p>
</div>
<div>
<p>Schweiger, O., Settele, J., Kudrna, O., Klotz, S., &amp; Kühn, I. (2008). Climate change can cause spatial mismatch of trophically interacting species. <em>Ecology</em>, <em>89</em>(12), 3472–3479.</p>
</div>
<div>
<p>Senthong, V., Wang, Z., Li, X. S., Fan, Y., Wu, Y., Wilson Tang, W., &amp; Hazen, S. L. (2016). Intestinal microbiota-generated metabolite trimethylamine-n-oxide and 5-year mortality risk in stable coronary artery disease: The contributory role of intestinal microbiota in a courage-like patient cohort. <em>Journal of the American Heart Association</em>, <em>5</em>(6), e002816.</p>
</div>
<div>
<p>Smith, M. I., Yatsunenko, T., Manary, M. J., Trehan, I., Mkakosya, R., Cheng, J., … others. (2013). Gut microbiomes of malawian twin pairs discordant for kwashiorkor. <em>Science</em>, <em>339</em>(6119), 548–554.</p>
</div>
<div>
<p>Smits, S. A., Leach, J., Sonnenburg, E. D., Gonzalez, C. G., Lichtman, J. S., Reid, G., … others. (2017). Seasonal cycling in the gut microbiome of the hadza hunter-gatherers of tanzania. <em>Science</em>, <em>357</em>(6353), 802–806.</p>
</div>
<div>
<p>Sokol, H., Pigneur, B., Watterlot, L., Lakhdari, O., Bermúdez-Humarán, L. G., Gratadoux, J.-J., … others. (2008). Faecalibacterium prausnitzii is an anti-inflammatory commensal bacterium identified by gut microbiota analysis of crohn disease patients. <em>Proceedings of the National Academy of Sciences</em>, <em>105</em>(43), 16731–16736.</p>
</div>
<div>
<p>Stelling, J., Sauer, U., Szallasi, Z., Doyle III, F. J., &amp; Doyle, J. (2004). Robustness of cellular functions. <em>Cell</em>, <em>118</em>(6), 675–685.</p>
</div>
<div>
<p>Sun, Y. V., &amp; Hu, Y.-J. (2016). Integrative analysis of multi-omics data for discovery and functional studies of complex human diseases. <em>Advances in Genetics</em>, <em>93</em>, 147–190.</p>
</div>
<div>
<p>Thaiss, C. A., Levy, M., Grosheva, I., Zheng, D., Soffer, E., Blacher, E., … others. (2018). Hyperglycemia drives intestinal barrier dysfunction and risk for enteric infection. <em>Science</em>, <em>359</em>(6382), 1376–1383.</p>
</div>
<div>
<p>Tims, S., Derom, C., Jonkers, D. M., Vlietinck, R., Saris, W. H., Kleerebezem, M., … Zoetendal, E. G. (2013). Microbiota conservation and bmi signatures in adult monozygotic twins. <em>The ISME Journal</em>, <em>7</em>(4), 707–717.</p>
</div>
<div>
<p>Tourassi, G. D., Frederick, E. D., Markey, M. K., &amp; Floyd Jr, C. E. (2001). Application of the mutual information criterion for feature selection in computer-aided diagnosis. <em>Medical Physics</em>, <em>28</em>(12), 2394–2402.</p>
</div>
<div>
<p>Uhler, C. (2017). Gaussian graphical models: An algebraic and geometric perspective. <em>arXiv Preprint arXiv:1707.04345</em>.</p>
</div>
<div>
<p>Villa-Vialaneix-nathalie, N. (n.d.). <em>An introduction to network inference and mining</em>.</p>
</div>
<div>
<p>Viniegra Velázquez, L. (2014). El reduccionismo cientı́fico y el control de las conciencias: Parte i. <em>Boletı́n Médico Del Hospital Infantil de México</em>, <em>71</em>(4), 252–257.</p>
</div>
<div>
<p>Von Neuman, J., Burks, A., Gardner, M., Wolfram, S., Wolfram, S., Sipper, M., … others. (1966). <em>Theory of self-reproducing automata</em>.</p>
</div>
<div>
<p>Wang, Y., Ghumare, E., Vandenberghe, R., &amp; Dupont, P. (2017). Comparison of different generalizations of clustering coefficient and local efficiency for weighted undirected graphs. <em>Neural Computation</em>, <em>29</em>(2), 313–331.</p>
</div>
<div>
<p>Watts, D. J., &amp; Strogatz, S. H. (1998). Collective dynamics of ‘small-world’networks. <em>Nature</em>, <em>393</em>(6684), 440–442.</p>
</div>
<div>
<p>Yatsunenko, T., Rey, F. E., Manary, M. J., Trehan, I., Dominguez-Bello, M. G., Contreras, M., … others. (2012). Human gut microbiome viewed across age and geography. <em>Nature</em>, <em>486</em>(7402), 222–227.</p>
</div>
<div>
<p>Zhang, B., &amp; Horvath, S. (2005). A general framework for weighted gene co-expression network analysis. <em>Statistical Applications in Genetics and Molecular Biology</em>, <em>4</em>(1).</p>
</div>
<div>
<p>Zheng, H., Wang, H., Dewhurst, R. J., &amp; Roehe, R. (2018). Improving the inference of co-occurrence networks in the bovine rumen microbiome. <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em>, <em>17</em>(3), 858–867.</p>
</div>
<div>
<p>Zheng, W., Wang, D., &amp; Zou, X. (2019). Control of multilayer biological networks and applied to target identification of complex diseases. <em>BMC Bioinformatics</em>, <em>20</em>(1), 1–12.</p>
</div>
<div>
<p>Zhu, W., Gregory, J. C., Org, E., Buffa, J. A., Gupta, N., Wang, Z., … others. (2016). Gut microbial metabolite tmao enhances platelet hyperreactivity and thrombosis risk. <em>Cell</em>, <em>165</em>(1), 111–124.</p>
</div>
<div>
<p>Zuzek, L. A., Stanley, H. E., &amp; Braunstein, L. A. (2015). Epidemic model with isolation in multilayer networks. <em>Scientific Reports</em>, <em>5</em>(1), 1–7.</p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="antecedentes.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/03-Materiales_y_metodos.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
